src/map/itemdb.cpp

```diff
@@ -1197,38 +1194,40 @@ int itemdb_searchname_array(struct item_data** data, int size, const char *str)
 	return count;
 }
 
+std::shared_ptr<s_item_group_entry> itemdb_get_random_itemsubgroup(std::shared_ptr<s_item_group_random> random) {
```

Comment:
Turn this into a function of s_item_group_random maybe?

Reply:
Hmm what do you mean?

Reply:
https://github.com/rathena/rathena/pull/6032#discussion_r666507757